timeline/style.css

```diff
@@ -0,0 +1,84 @@
+.timeline-line:after {
+  content: "";
+  position: absolute;
+  width: 2px;
+  height: 100%;
+  background: #dddddd;
+  z-index: 1;
+  left: 16px;
+}
+.space-marker {
```

Comment:
.timeline-group .row:last-child .timeline {
  display: none
}
can i use this?